app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is the list of users</h1>
```

Comment:
- [ ] Add a `show` file that corresponds to the show method in the `users_controllers`